taxi/forms.py

```
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.core.validators import MinValueValidator, RegexValidator
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class CarCreationForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = "__all__"
+
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
+        widget=forms.CheckboxSelectMultiple,
+        required=False,
+    )
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data.get("license_number")
+        if not RegexValidator(
```

RegexValidator is used incorrectly here. It should be called directly and will raise a ValidationError if the value is invalid. The current usage as a boolean check is incorrect. This violates the checklist item requiring proper license number validation.